hello-world-mtls/README.md

```diff
@@ -1,11 +1,119 @@
-# Hello World with mTLS
+# Hello World with TLS
 
-This example shows how to secure your Temporal application with [mTLS](https://docs.temporal.io/security/#encryption-in-transit-with-mtls).
-This is required to connect with Temporal Cloud or any production Temporal deployment.
+This sample demonstrates how to secure your Temporal application with in-transit TLS encryption.
 
-### Running this sample
+This is required to connect with [Temporal Cloud](https://docs.temporal.io/cloud/security#encryption),
+or to a self-hosted Temporal deployment that is [secured with TLS](https://docs.temporal.io/self-hosted-guide/security#encryption-in-transit-with-mtls).
 
-See the [mTLS tutorial](https://legacy-documentation-sdks.temporal.io/typescript/security/#mtls-tutorial).
+## Running this sample
 
-On step 8, run `npm run start` to start the Worker.
-On step 9, in another shell, enter `npm run workflow` to run the Workflow. It should print out `Hello, Temporal!`
+1. Install NPM dependencies:
+
+   ```
+   npm install   # or `pnpm` or `yarn`
+   ```
+
+2. Set and export the following environment variables, as appropriate for your Temporal cluster.
+
+   - `TEMPORAL_ADDRESS` — gRPC endpoint of your Temporal cluster
+   - `TEMPORAL_NAMESPACE` — Your namespace
+
+   - `TEMPORAL_SERVER_ROOT_CA_CERT_PATH` — Path to a file containing the Root CA certificate
+     to use to validate your _server_'s certificate. Omit to use CA certificates configured
+     on your operating system.
+   - `TEMPORAL_SERVER_NAME_OVERRIDE` — Set to override the target name (SNI) used for TLS host
+     name checking. This can be useful when you have reverse proxy in front of temporal server,
+     and you may want to override the SNI to direct traffic to the appropriate backend server
+     based on custom routing rules. Oppositely, connections could be refused if the provided
+     SNI does not match the expected host. Adding this override should be done with care.
+
+   _(for mTLS authentication only)_
+
+   - `TEMPORAL_CLIENT_CERT_PATH` — Path to a file containing your client certificate
+   - `TEMPORAL_CLIENT_KEY_PATH` — Path to a file containing your client private key
+
+   _(for API Key authentication only)_
+
+   - `TEMPORAL_CLIENT_API_KEY` — The API Key used to identify this Client or Worker.
+
+   Refer to the [Common Configurations](#common-configurations) section below for details.
+
+3. Run the Worker:
+
+```
+npm run start.watch` to start the Worker
```

Comment:
extra backtick after `watch`, or is that intended to show the end of the command?

Reply:
Thanks, good catch!